README.md

```diff
@@ -101,10 +101,12 @@ ardrive upload-file --wallet-file /path/to/my/wallet.json --parent-folder-id "f0
         1. [Uploading a Single File](#uploading-a-single-file)
         2. [Download a Single File (BETA)](#download-file)
         3. [Uploading a Folder with Files](#bulk-upload)
-        4. [Uploading Multiple Files](#multi-file-upload)
-        5. [Fetching the Metadata of a File Entity](#fetching-the-metadata-of-a-file-entity)
-        6. [Uploading Manifests](#uploading-manifests)
-        7. [Hosting a Webpage with Manifest](#hosting-a-webpage-with-manifest)
+        4. [Downloading a Folder with Files](#download-folder)
+        5. [Uploading Multiple Files](#multi-file-upload)
+        6. [Fetching the Metadata of a File Entity](#fetching-the-metadata-of-a-file-entity)
+        7. [Uploading Manifests](#uploading-manifests)
+        8. [Hosting a Webpage with Manifest](#hosting-a-webpage-with-manifest)
+        9. [Create New Drive and Upload Folder Pipeline Example](#create-upload-pipeline)
```

Comment:
This section `create-upload-pipeline` was removed on dev due to incompatibility with drive bundles. Probably showing up here from a merge conflict.

Reply:
It was because a wrong merge conflict resolution on my side. It's removed at e5bb0b9353f696fc3b526bf066a433c107967e70